sphinx/pycode/ast.py

```diff
@@ -187,3 +187,9 @@ def visit_Tuple(self, node: ast.Tuple) -> str:
 
     def generic_visit(self, node: ast.AST) -> NoReturn:
         raise NotImplementedError('Unable to parse %s object' % type(node).__name__)
+
+    def visit_Slice(self, node: ast.Slice) -> str:
```

Comment:
Can you put the method above in alphabetical order ? (I think the `visit_*` methods are alphabetically ordered).